src/core/mac/channel_mask.hpp

```diff
@@ -61,7 +62,7 @@ namespace Mac {
  * It is a wrapper class around a `uint32_t` bit vector representing a set of channels.
  *
  */
-class ChannelMask
+class ChannelMask : public NotEquatable<ChannelMask>
```

Comment:
nit: the name `NotEquatable` is a little bit odd because `ChannelMask` objects are equitable but it inherited from `NotEquatable<ChannelMask>`. It sounds like that we can't use `==` and `!=` on `ChannelMask` objects. 

Maybe rename to `Unequatable` ?

```suggestion
class ChannelMask : public Unequatable<ChannelMask>
```

Reply:
Thanks @simonlingoogle.  I definitely do like the `Unequatable` more (was not very fond of `NotEquatable` name).